src/main/webapp/app/shared/sidebar/sidebar.component.scss

```diff
@@ -42,3 +42,7 @@
         display: none !important;
     }
 }
+
+.link-primary {
```

Comment:
I want the `Filter` option to be highlighted blue, if we remove this class it would look like in the screencast below:

![filterNotHighlightedBlue](https://github.com/user-attachments/assets/fdfd152d-f271-448e-b5b7-8f07c3447604)

Reply:
Ah, the next suggestion explains it, thanks! https://github.com/ls1intum/Artemis/pull/8858#discussion_r1683110838